resources/views/admin/user.blade.php

```diff
@@ -354,17 +354,17 @@
             </tr>
             <tr class="trodd">
                 <td id="nob">
-                    <a href="/authtokens/manage">Manage authentication tokens</a>
+                    <a href="authtokens/manage">Manage authentication tokens</a>
                 </td>
             </tr>
             <tr class="treven">
                 <td id="nob">
                     <a href="upgrade.php">Maintenance</a>
                 </td>
             </tr>
-            <tr class="trodd">
+            <tr class="trodd" ng-if="::cdash.show_monitor">
```

Comment:
If this condition is false, there will be two `treven` rows next to each other, which will break the striped table format.  The proper way to do this is to get rid of the `treven`/`trodd` stuff altogether, but I can do that as part of a broader refactor later.  For now, the best solution might be to use `ng-class-even` and `ng-class-odd` to make sure the table is striped appropriately.